corehq/apps/hqadmin/management/commands/cchq_prbac_bootstrap.py

```diff
@@ -161,7 +161,11 @@ def remove_grant(self, priv_slug, dry_run=False):
         Role(slug=privileges.ADVANCED_DOMAIN_SECURITY, name='Advanced Domain Security',
              description='Allows domains to set security policies for all web users'),
         Role(slug=privileges.BUILD_PROFILES, name='Application Profiles',
-             description='Allows domains to create application profiles to customize app deploys')
+             description='Allows domains to create application profiles to customize app deploys'),
+        Role(slug=privileges.EXCEL_DASHBOARD, name="Excel Dashbord",
+             description="Allows domains to create Excel dashboard html exports"),
```

Comment:
_Supernit_: Would I be going too far if I suggested we standardize on English style conventions, like always spelling "Excel" with a capital "E", and always spelling "HTML" in caps (unless we mean "to be outstandingly skillful", or we are referring to the traditional Czechoslovakian porridge, "htmŀ")?

(Yeah, so I totally made up the thing about the porridge. :stuck_out_tongue: )